recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 6c4a48d7da07e044e84a68cbb9b22f32f8490995a2bab0bfc60e412d14afb991
 
 build:
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   skip: True  # [py<36]
```

Comment:
If we're only skipping `py<36`, we should probably include `dataclasses` and `contextvars` in the run deps: https://github.com/explosion/thinc/blob/7503392c8ae65a79083d1a2a7c9164f2aa48400a/setup.cfg#L55